### ISSUE TITLE
Nonlinear regression analysis (logit, tobit and probit)

### ISSUE THREAD

**Issue by marsscpo** [jasp-stats/jasp-desktop#660]
Hi, 

OLS regression is not fit for variables that are not normally distributed, e.g. dummy or censored variables.

Since many variables of interest fall into that category, it would be very useful to have logit, tobit and probit functions in addition to OLS.

Cheers!

**Comment by jonathon-love** [comment]
yup, we have these tentatively planned for the 0.8

**Comment by marsscpo** [comment]
Great, thanks!

Any idea when the 0.8 might be available?

**Comment by jonathon-love** [comment]
at a guess, i would say in three months time.

**Comment by HarperAnalytics** [comment]
Great software packages. Any update on when non-linear regression will be updated? I'd like to share with some colleagues in public health but it has less use for public health applications without implementation of logit regression?

**Comment by quadrant561** [comment]
Bump. Logistic regression would be amazing!

**Comment by MatthiasTreder** [comment]
I second that. I'd like to teach JASP for medical research but I can't due to the lack of logistic regression. It would greatly expand the user base (much more than yet another Bayesian tool :P )

Any plans to implement it soon? If not, I might try to give it a go myself.

**Comment by NeedlessToSay** [comment]
I second that. I'd like to teach JASP for medical research but I can't due to the lack of logistic regression. It would greatly expand the user base (much more than yet another Bayesian tool :P )

Any plans to implement it soon? If not, I might try to get into the code & give it a go myself.

**Comment by petestmarie** [comment]
Another voice for the addition of logistic regression. In the context of an academic medical center, this would be the perfect analysis environment for smaller research projects that don't receive full support from our epidemiology/biostatistics group.

**Comment by asaji** [comment]
Any updates on whether logistic regression will be added?

**Comment by EJWagenmakers** [comment]
Yes! Erik-Jan van Kesteren has a first version working. We will have to ask
feedback and fine-tune it, but I am already quite pleased with the result.
It will ship in one of the next versions.

Cheers,
E.J.

********************************************
Eric-Jan Wagenmakers
Department of Psychological Methods, room G 0.29
University of Amsterdam, Nieuwe Achtergracht 129B
Letter: PO Box 15906, 1001 NK Amsterdam
Parcel: Valckenierstraat 59,  1018 XE Amsterdam

Web: ejwagenmakers.com
Book: bayesmodels.com
Stats: jasp-stats.org
Email: EJ.Wagenmakers@gmail.com
Cell: (+31) 6 45626624

“Man follows only phantoms.”
Pierre-Simon Laplace, last words
********************************************

**Comment by bramzandbelt** [comment]
I was thinking of implementing logistic regression using the Savage-Dickey method. Good to hear that you have a first version working already. Any chance I can help testing it?

**Comment by EJWagenmakers** [comment]
Hi Bram! @vankesteren 

Classical logistic regression only awaits testing -- you are more than welcome to help out. With respect to the Bayesian equivalent, please PM me for details.

Cheers,
E.J.
